src/config/dataset-formats/xlsx.md

```diff
@@ -0,0 +1,23 @@
+# Xlsx(Microsoft Excel)
+
+Thanks to [calamine](https://github.com/tafia/calamine) crate, ROAPI supports the `.xlsx` file.
```

Comment:
I don't think we need this reference line here (could get stale at some point). Could you please remove?